apps/web/src/design-system/template-button/DragButton.tsx

```diff
@@ -31,9 +33,11 @@ export function DragButton({ description, label, Icon }: IDragButtonProps) {
             </IconWrapper>
             <StyledContentWrapper>
               <Text weight="bold">{label}</Text>
-              <Text mt={3} color={colors.B60}>
-                {description}
-              </Text>
+              <When truthy={description.length > 0}>
```

Comment:
Not critical but can easily be: 
```suggestion
              <When truthy={description}>
```